packages/lit-dev-content/site/tutorials/content/reactivity/03.md

```diff
@@ -0,0 +1,27 @@
+Setting a reactive property or calling `requestUpdate()` triggers a reactive update cycle. At the beginning of an update cycle, Lit calls the `shouldUpdate()` method, passing in a map of changed properties.
+
+You can override `shouldUpdate()` to determine whether the component needs to update itself.
```

Comment:
Let's explain that by default this returns true.

Reply:
Done.